ingestion/setup.py

```diff
@@ -224,6 +224,7 @@
     "elasticsearch": {
         VERSIONS["elasticsearch8"],
     },  # also requires requests-aws4auth which is in base
+    "exasol": {"sqlalchemy_exasol>=5"},
```

Comment:
not sure if we could pin this as `sqlalchemy_exasol~=5.0` to avoid it automatically going into 6.X and things breaking down.

Reply:
Good point! If it’s alright with you, I’d use `"sqlalchemy_exasol>=5,<6"` as constraint.

This approach is more flexible, particularly in scenarios where multiple major versions are compatible as dependency. I’m already aware that a major version change in `sqlalchemy_exasol` is on the horizon, which will be compatible for all users of the websocket-based sqla driver. However, this transition will necessitate a major version upgrade due to the breaking compatibility with legacy drivers.